### DISCUSSION TITLE
My code keeps blank it doesn't return anything please help

### DISCUSSION THREAD

**Discussion by juned-adenwalla** [withcatai/node-llama-cpp#201] [Q&A] [answered]
Hiii all I am using the code below after executing the file its completely blank as attached in screenshot, please help me solve this issue as I am stuck for more than 3 days now apparently if I run this through CLI it works from this command : 

npx --no node-llama-cpp chat --model .\models\phi-2-orange.Q2_K.gguf

But if I use the code shared below its blank I have given the screenshot for the same if anyone can help would be highly appreciated.

![image](https://github.com/withcatai/node-llama-cpp/assets/81737295/27d96743-098a-45db-aecc-b0434814526d)

```
import { fileURLToPath } from "url";
import path from "path";
import { getLlama, LlamaChatSession } from "node-llama-cpp";

try {
    const __dirname = path.dirname(fileURLToPath(import.meta.url));

    const llama = await getLlama({
        gpu: false
    });

    const model = await llama.loadModel({
        llama,
        modelPath: path.join(__dirname, "models", "phi-2-orange.Q2_K.gguf")
    });

    const context = await model.createContext({
        model,
        contextSize: Math.min(4096, model.trainContextSize)
    });

    const session = new LlamaChatSession({
        contextSequence: context.getSequence()
    });

    const q1 = "Hi there, how are you?";
    console.log("User: " + q1);

    const a1 = await session.prompt(q1);
    console.log("AI: " + a1);

    const q2 = "Summarize what you said";
    console.log("User: " + q2);

    const a2 = await session.prompt(q2);
    console.log("AI: " + a2);

} catch (error) {
    console.error("Error occurred:", error);
}
```

**Comment by giladgd** [comment, marked as answer]
As I mentioned in my [other comment](https://github.com/withcatai/node-llama-cpp/discussions/195#discussioncomment-9104677), there aren't enough details here for me to be able to help you, please provide the information I mentioned in that comment so I can help you.

If you say that the `chat` command worked for you while using your code with `gpu: false` didn't, I suspect that the `chat` command utilized some GPU layer on your computer which made inference be much faster, and using the CPU only (with `gpu: false` in your code) made the inference so slow you thought nothing happened and that it's stuck, while it's actually doing the inference but very very slowly
